src/server/game/Entities/Creature/GossipDef.cpp

```diff
@@ -319,12 +319,10 @@ void PlayerMenu::SendQuestGiverQuestList(QEmote const& eEmote, std::string const
 
     if (QuestGreeting const* questGreeting = sObjectMgr->GetQuestGreeting(guid.GetTypeId(), guid.GetEntry()))
     {
-        std::string strGreeting = questGreeting->Text;
-
-        LocaleConstant localeConstant = _session->GetSessionDbLocaleIndex();
-        if (localeConstant != LOCALE_enUS)
-            if (QuestGreetingLocale const* questGreetingLocale = sObjectMgr->GetQuestGreetingLocale(guid.GetTypeId(), guid.GetEntry()))
-                ObjectMgr::GetLocaleString(questGreetingLocale->Greeting, localeConstant, strGreeting);
+        LocaleConstant locale = _session->GetSessionDbLocaleIndex();
+        std::string strGreeting = questGreeting->Greeting[DEFAULT_LOCALE];
```

Comment:
@Kitzunu this is crashing the server

https://gist.github.com/Nyeriah/11c0553ed968884cfe96b3277c9dbb80